docs/antrea-ipam.md

```diff
@@ -31,7 +31,8 @@ NodeIPAM dictionary contains the following items:
 controller. Default is false.
 
 - `clusterCIDRs`: CIDR Ranges for Pods in cluster. String array containing single
-CIDR range, or multiple ranges. The CIDRs could be either IPv4 or IPv6.
+CIDR range, or multiple ranges. The CIDRs could be either IPv4 or IPv6. Example
+values: `[172.100.0.0/16]`, `[172.100.0.0/20, 172.100.32.0/20, fd00:172:100::/64]`.
```

Comment:
Sorry for the late review. I think multiple ranges per IP family is not supported by NodeIPAM. I tested it to confirm:
```
range_allocator.go:389] Failed to update node k8s04 PodCIDR to [192.168.3.0/24 192.169.0.0/24 fd00:172:100::/64] after multiple attempts: failed to patch node CIDR: Node "k8s04" is invalid: spec.podCIDRs: Invalid value: []string{"192.168.3.0/24", "192.169.0.0/24", "fd00:172:100::/64"}: may specify no more than one CIDR for each IP family
```

Reply:
Ah, I just tested NodeIPAM can be initialized with multiple CIDRs. Thanks for the fix.